types/crypto-pouch/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for crypto-pouch 4.0
+// Project: https://github.com/calvinmetcalf/crypto-pouch
+// Definitions by: Franklin Simmons <https://github.com/FranklinSimmons>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="pouchdb-core" />
+
+declare namespace PouchDB {
+    namespace CryptoPouch {
+        interface Options {
+            /* A string password, used to encrypt documents. */
+            password: string;
+            /* Array of strings of properties that will not be encrypted. */
+            ignore?: string[];
+        }
+    }
+
+    interface Database<Content extends {} = {}> {
+        /**
+         *
+         * @param options see CryptoPouch.Options
+         */
+        crypto(options: CryptoPouch.Options): Promise<void>;
+        /**
+         *
+         * @param password A string password, used to encrypt documents.
+         * @param ignore Array of strings of properties that will not be encrypted.
+         */
+        crypto(password: string, ignore?: string[]): Promise<void>;
+        /**
+         *
+         * Disables encryption on the database and forgets your password.
+         */
+        removeCrypto(): void;
+    }
+}
+
+declare module 'crypto-pouch' {
+    const plugin: PouchDB.Plugin;
+    export = plugin;
+}
```

Comment:
I think you can use newer syntax and avoid turning off lint rules like this:

```ts
import PouchDB = require('pouchdb');
declare module "pouchdb" {
  declare namespace PouchDB {
    // lines 9-35 go here ...
  }
}
const Plugin: PouchDB.Plugin;
export = Plugin;
export as namespace Plugin;
```

This
1. Imports PouchDB
1. Augments a namespace PouchDB inside the "pouchdb" module.
3. Re-exports PouchDB.Plugin
4. Makes PouchDB.Plugin globally available as `Plugin`.

Reply:
@sandersn thank you for reviewing this pull request and the suggestion.  I can't quite go the route you suggested however and I believe it is due to `PouchDB` being a declared global in [pouchdb-core/index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/pouchdb-core/index.d.ts)

```
declare module 'pouchdb-core' {
  const PouchDb: PouchDB.Static;
  export = PouchDb;
}

declare var PouchDB: PouchDB.Static;
```
So it would appear the correct definition should be
```
import PouchDB = require('pouchdb');
declare global {
    namespace PouchDB {
    /// gore
   }
}    
declare const Plugin: PouchDB.Plugin;
export = Plugin;
```
As PouchDB is global, I didn't see the utility in #4 of your point list, if I understand the gist of it.  If this sounds right I'll commit the changes - thank you very much for lending your expertise.